### PR TITLE
[ASSIST-429]: UI/UX: Artificial Intelligence Render Formatted Text 

### PR DESCRIPTION
**Pull request by Orrison**
Set's up the process to display formatted AI text via markdown that is sanitized.

During this process, I have started the first implementation of how we should be setting up JS implementations that rely on NPM packages.
@danharrin let me know if you think there is a better way.
The important things are that it is compiled so that we can make use of NPM packages without having to load them directly onto the pages. It is easy to configure. And that the JS is only ever loaded on the page it is needed.

### CODE REVIEW

**Review by danharrin — APPROVED** [review]
Might be worth reformatting the Blade as there were some indentation changes

File: app-modules/assistant/resources/views/filament/pages/ai-assistant.blade.php [hunk 1]
```diff
@@ -43,64 +47,64 @@
...
                     @if (!$chat->id)
                         <button
-                            class="inline-flex cursor-pointer justify-center rounded-lg p-2 text-gray-500 hover:bg-gray-100 hover:text-gray-900 dark:text-gray-400 dark:hover:bg-gray-600 dark:hover:text-white"
-                            type="button"
-                            wire:loading.attr="disabled"
-                            wire:click="save"
+                                class="inline-flex cursor-pointer justify-center rounded-lg p-2 text-gray-500 hover:bg-gray-100 hover:text-gray-900 dark:text-gray-400 dark:hover:bg-gray-600 dark:hover:text-white"
+                                type="button"
+                                wire:loading.attr="disabled"
+                                wire:click="save"
```
Comment (danharrin) [inline app-modules/assistant/resources/views/filament/pages/ai-assistant.blade.php:59]: Is this correct?

Comment (Orrison) [reply app-modules/assistant/resources/views/filament/pages/ai-assistant.blade.php:59]: Do you mean the formatting, or something else?

Comment (danharrin) [reply app-modules/assistant/resources/views/filament/pages/ai-assistant.blade.php:59]: These changes indent the attributes an extra level, I didn't think this is how HTML was formatted in this app

**Review by danharrin — APPROVED** [review]

**Pull request merged**